app/views/controllers/users/show/_profile.erb

```diff
@@ -63,7 +63,13 @@ footer =
       str, url, args = tab
       args ||= {}
       kwargs = merge_tab_args_with_extra_args(args, { class: "d-block" }) %>
-      <%= link_to(str, url, kwargs) %>
+      <% if args.any?([:button, :destroy]) %>
+        <%= button_to(:destroy_object.t(type: :user), url, method: :delete,
+                      class: "d-block",
+                     ) %>
+      <% else %>
+        <%= link_to(str, url, kwargs) %>
+      <% end %>
```

Comment:
@JoeCohen I think you want the `destroy_button` helper here, from `helpers/link_helper.rb`. 
That's what's used everywhere else.